src/components/AccountForm/AccountDetails/AccountDetails.css

```diff
@@ -0,0 +1,26 @@
+.account-details-container {
```

Comment:
I noticed in your demo that the buttons did edit account * buttons did not stretch to the size of the container. Was that resolved?